### PR TITLE
Windows build

### PR DESCRIPTION
**Pull request by rkimballn1**
With this you can build nbench with the INTERPRETER backend. You can build unit test too but I did not run those tests.

Here are the commands to build:
```
cmake ../ngraph -DNGRAPH_CPU_ENABLE=FALSE  -G "Visual Studio 15 2017 Win64"
cmake --build . --target nbench --config Release
```

The `-G` option is specified so we can build 64 bit. 32 bit is the default :(

### CODE REVIEW

**Comment by shssf** [comment]
@rkimballn1 Is it possible to create simple build scripts in the project root?
For example as it is in https://github.com/intel/clDNN (`create_unixmake_gcc.sh` for Linux build and `create_msvc_mscc.bat` for Windows)

File: src/ngraph/runtime/reference/topk.hpp [hunk 1]
```diff
@@ -28,6 +28,17 @@ namespace ngraph
     {
         namespace reference
         {
+            template <typename T, typename U>
```
Comment (diyessi) [inline src/ngraph/runtime/reference/topk.hpp]: Why are these changed?

File: test/util/test_tools.cpp [hunk 2]
```diff
@@ -31,57 +31,90 @@ vector<float> read_float_vector(shared_ptr<runtime::Tensor> tv)
     if (element_type == element::boolean)
     {
         vector<char> vec = read_vector<char>(tv);
-        float_vec = vector<float>(vec.begin(), vec.end());
+        for (char value : vec)
```
Comment (diyessi) [inline test/util/test_tools.cpp]: Why are these changes here?

**Comment by jbobba** [comment]
Are we going to enable Windows CI builds to keep this from breaking?

**Comment by rkimballn1** [comment]
Closing but not deleting branch. This PR is too large and must be split into multiple smaller PRs. The first of these is #2177
